TeamCode/src/main/java/org/firstinspires/ftc/teamcode/systems/imu/IMUSystem.java

```diff
@@ -111,5 +111,9 @@ String formatAngle(AngleUnit angleUnit, double angle) {
     String formatDegrees(double degrees) {
         return String.format(Locale.getDefault(), "%.1f", AngleUnit.DEGREES.normalize(degrees));
     }
+
+    public void reIninitalize() {
+        imu.initialize(parameters);
```

Comment:
What is this for? If you need parameters for the IMU shouldn't that be done in the constructor?

Reply:
That is my attempt to recalibrate the IMU, but I don't think it can actually be done by re initializing it, so I will delete this